.github/workflows/ci.yml

```diff
@@ -16,11 +16,19 @@ jobs:
 
     steps:
 
-    - name: Set up JDK
+    - name: Set up JDK for build and test
+      if: matrix.jdk != 17 
```

Comment:
Does JDK17 not support Gradle 6.9?

Reply:
No it does not

Reply:
This was followed in https://github.com/opensearch-project/common-utils/pull/121